src/displayapp/widgets/StatusIcons.cpp

```diff
@@ -37,6 +43,8 @@ void StatusIcons::Update() {
     batteryIcon.SetBatteryPercentage(batteryPercent);
   }
 
+  lv_obj_set_hidden(alarmIcon, (alarmController.State() == Pinetime::Controllers::AlarmController::AlarmState::Not_Set));
```

Comment:
I would put the equality in its own line and give it a meaningful name.

Reply:
Is it better now?

Reply:
Yes :)